File: Data Structures & Algorithms/majority-element/submission-1.java
class Solution {
    public int majorityElement(int[] nums) {
        HashMap<Integer, Integer> count = new HashMap<>();
        int res = 0, countMax = 0;

        for (int num : nums) {
            count.put(num, count.getOrDefault(num, 0) + 1);
            if (count.get(num) > countMax) {
                res = num;
                countMax = count.get(num);
            }
        }
        return res;
    }
}

File: Data Structures & Algorithms/max-consecutive-ones/submission-1.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        max1, score = 0, 0
        for num in nums:
            if num == 1:
                score += 1 
            else:
                max1 = max(score, max1)
                score = 0
        return max(score, max1)
        